views/account/change_login.erb

```diff
@@ -16,6 +16,9 @@
               <%== rodauth.change_login_additional_form_tags %>
               <%== rodauth.csrf_tag("/" + rodauth.change_login_route) %>
               <div class="mt-6 grid grid-cols-6 gap-6">
+                <div class="col-span-6 sm:col-span-3 xl:col-span-2">
+                  <%== render("components/form/text", locals: {label: "Current Email", value: @current_user.email, extra_class: "bg-gray-50 text-gray-500 ring-gray-200 ", attributes: {disabled: true}}) %>
```

Comment:
This doesn't need to be input, right? It is disabled anyway. We can use just plain text.

Reply:
Sure, we can do that. However, when we use plain text, we have to manually add a label and styles. Using disabled text input was simpler because it automatically aligns with the current items.

<img width="1508" alt="Screenshot 2024-02-12 at 12 50 13" src="https://github.com/ubicloud/ubicloud/assets/993199/84f8aeef-9b95-4bda-bcfa-3d5bbfcbc8d7">